flume/doc/design/OrionTestSink.md

```diff
@@ -0,0 +1,23 @@
+# OrionTestSink
```

Comment:
If the refactoring task is pending for the future, I'd suggest to create an issue for that.